modules/pytket-cirq/tests/test_cirq_backend.py

```diff
@@ -17,19 +17,18 @@
 import math
 from hypothesis import given, strategies
 import numpy as np
-from pytket.extensions.cirq import (
+from pytket.extensions.cirq.backends.cirq import (  # type: ignore
```

Comment:
same here, sorry for bugging you on this. I'm sure there is a one-line answer to my annoyingness, at least.